bundles/org.openhab.binding.phc/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -94,25 +88,25 @@
 			<parameter name="upDownTime1" type="integer" min="1" max="65535">
 				<advanced>true</advanced>
 				<label>Time Shutter 1</label>
-				<description>The Time which the first shutter needs to move up/down.</description>
+				<description>The time (in seconds) which the first shutter needs to move up/down.</description>
```

Comment:
You can add the attribute `unit="s"` to the `parameter element to specify this is in seconds. Then it's not needed to add the unit in the description. But you can change that in some future improvement.